usaspending_api/database_scripts/matview_generator/universal_transaction_matview.json

```diff
@@ -255,153 +199,14 @@
-    }, {
-      "name": "business_categories",
-      "method": "GIN",
-      "columns": [{"name": "business_categories"}]
-    }, {
-      "name": "keyword_ts_vector",
-      "method": "GIN",
-      "columns": [{"name": "keyword_ts_vector"}]
-    }, {
-      "name": "award_ts_vector",
-      "method": "GIN",
-      "columns": [{"name": "award_ts_vector"}]
-    }, {
-      "name": "recipient_name_ts_vector",
-      "method": "GIN",
-      "columns": [{"name": "recipient_name_ts_vector"}]
-    }, {
-      "name": "treasury_account_identifiers",
-      "method": "GIN",
-      "columns": [{"name": "treasury_account_identifiers", "opclass": "gin__int_ops"}]
     }, {
       "name": "simple_pop_geolocation",
       "where": "pop_country_code = 'USA' AND pop_state_code IS NOT NULL AND action_date >= '2007-10-01'",
       "columns": [{"name": "pop_state_code"}, {"name": "action_date"}]
-    }, {
-      "name": "compound_geo_pop_1",
-      "where": "pop_country_code = 'USA' AND pop_state_code IS NOT NULL AND action_date >= '2007-10-01'",
-      "columns": [{"name": "pop_state_code"}, {"name": "pop_county_code"}, {"name": "action_date"}]
-    }, {
-      "name": "compound_geo_pop_2",
-      "where": "pop_country_code = 'USA' AND pop_state_code IS NOT NULL AND action_date >= '2007-10-01'",
-      "columns": [{"name": "pop_state_code"}, {"name": "pop_congressional_code"}, {"name": "action_date"}]
-    }, {
-      "name": "compound_geo_pop_3",
-      "where": "pop_country_code = 'USA' AND pop_zip5 IS NOT NULL AND action_date >= '2007-10-01'",
-      "columns": [{"name": "pop_zip5"}, {"name": "action_date"}]
-    }, {
-      "name": "simple_recipient_location_geolocation",
-      "where": "recipient_location_country_code = 'USA' AND recipient_location_state_code IS NOT NULL AND action_date >= '2007-10-01'",
-      "columns": [{"name": "recipient_location_state_code"}, {"name": "action_date"}]
-    }, {
-      "name": "compound_geo_rl_1",
-      "where": "recipient_location_country_code = 'USA' AND recipient_location_state_code IS NOT NULL AND action_date >= '2007-10-01'",
-      "columns": [{"name": "recipient_location_state_code"}, {"name": "recipient_location_county_code"}, {"name": "action_date"}]
-    }, {
-      "name": "compound_geo_rl_2",
-      "where": "recipient_location_country_code = 'USA' AND recipient_location_state_code IS NOT NULL AND action_date >= '2007-10-01'",
-      "columns": [{"name": "recipient_location_state_code"}, {"name": "recipient_location_congressional_code"}, {"name": "action_date"}]
-    }, {
-      "name": "compound_geo_rl_3",
-      "where": "recipient_location_country_code = 'USA' AND recipient_location_zip5 IS NOT NULL AND action_date >= '2007-10-01'",
-      "columns": [{"name": "recipient_location_zip5"}, {"name": "action_date"}]
     }, {
       "name": "recipient_hash",
       "where": "action_date >= '2007-10-01'",
       "columns": [{"name": "recipient_hash"}]
-    }, {
-      "name": "type_pre2008",
```

Comment:
Shouldn't `type_pre2008` remain for downloads using the required Award Type filter? Custom Award Downloads go back to FY2001

![Screen Shot 2020-07-25 at 7 31 23 AM](https://user-images.githubusercontent.com/22781949/88458134-13e60e80-ce49-11ea-972c-b3b8ef39e77d.png)

Reply:
@tony-sappe when doing before and after explain plans, it looks like the **action_date_pre2008** index was used for both. I can add **award_type_pre2008** back in if you think it would be safer though.

Reply:
> before and after explain plans

With what filters?

Reply:
Seems OK